pkg/storage/couchdb/couchdbstore_test.go

```diff
@@ -218,6 +218,20 @@ func TestCouchDBStore_Put(t *testing.T) {
 		err := store.Put(testDocKey, []byte(testNonJSONValue))
 		require.NoError(t, err)
 	})
+	t.Run("Success: put after delete", func(t *testing.T) {
```

Comment:
@sandrask is this what you meant?

Reply:
This is a good test - it tests your new error.